src/Compilers/CSharp/Portable/Symbols/Source/SourceUserDefinedOperatorSymbolBase.cs

```diff
@@ -13,6 +13,7 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     internal abstract class SourceUserDefinedOperatorSymbolBase : SourceOrdinaryMethodOrUserDefinedOperatorSymbol
     {
+        // tomat: ignoreDynamic should be true, but we don't want to introduce breaking change. See bug 605326.
```

Comment:
>605326 [](http://example.com/codeflow?start=104&length=6)

Is this a devdiv bug or a github bug?

Reply:
> Is this a devdiv bug or a github bug?

This comment was moved from elsewhere. I didn't try to "bind" the number.